packages/trpc/server/routers/viewer/eventTypes/update.handler.ts

```diff
@@ -474,6 +489,52 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
     }
   }
 
+  // Logic for updating `fieldTranslations`
+  // user has no description translations OR user is changing the description
+  const descriptionTranslationsNeeded =
+    eventType.fieldTranslations.filter((trans) => trans.field === EventTypeAutoTranslatedField.DESCRIPTION)
+      .length === 0 || newDescription;
+  const description = newDescription ?? eventType.description;
+
+  if (
+    ctx.user.organizationId &&
+    autoTranslateDescriptionEnabled &&
+    descriptionTranslationsNeeded &&
+    description
+  ) {
+    const userLocale = ctx.user.locale;
+
+    // Use process.nextTick to handle translations in the background
+    process.nextTick(async () => {
+      try {
+        // TODO: we want to support the other locales in Locales enum
+        const targetLocales = (["en", "es", "de", "pt", "fr", "it", "ar", "ru", "zh-CN"] as const).filter(
```

Comment:
"ru" doesn't exist in the i18n.json file. Does this throw an exception? It doesn't when running the CLI locally.
```suggestion
        const targetLocales = (["en", "es", "de", "pt", "fr", "it", "ar", "zh-CN"] as const).filter(
```